WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsFollowPresenter.swift

```diff
@@ -15,7 +15,7 @@ class ReaderCommentsFollowPresenter: NSObject {
 
     private let post: ReaderPost
     private weak var delegate: ReaderCommentsFollowPresenterDelegate?
-    private let presentingViewController: UIViewController
+    private unowned let presentingViewController: UIViewController
```

Comment:
Did you chose `unowned` so the type didn't have to become wrapped in `Optional`?

Reply:
Indeed. Given this is a "presenter" type, it should not use the view controller reference if the view controller is released. If this unowned reference causes a runtime crash in the production app, I think we may have other issues.